STAN.CLIENT/StanMsg.cs

```diff
@@ -22,6 +22,28 @@ public sealed class StanMsg
         internal MsgProto proto;
         private  AsyncSubscription sub;
 
+        /// <summary>
+        ///  Constructor for generating a StanMsg object.  Used only for application unit testing.
+        /// </summary>
+        /// <remarks>
+        /// Objects of this type are normally generated internally by the NATS streaming client.
+        /// This constructor has been provided to facilitate application unit testing.
+        /// </remarks>
+        /// <param name="data">The message payload.</param>
+        /// <param name="redelivered">True if the message may have been redelivered.</param>
+        /// <param name="subject">Subject of the message</param>
+        /// <param name="timestamp">Message timestamp, nanoseconds since epoch (1/1/1970)</param>
+        /// <param name="subscription">Subscription of the message.  May be null</param>

```

Comment:
This should probably be noted that it needs to be an `AsyncSubscription`.

Reply:
Good point...  I'll add it.  I also forgot sequence number.